### PR TITLE
[plugin.video.media-ccc-de] v0.1.1

### PR DESCRIPTION
**Pull request by TobiX**
### Description
- Supports multiple translated streams (if available)
### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.media-ccc-de/addon.xml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.media-ccc-de" name="media.ccc.de" version="0.1.0" provider-name="Tobias Gruetzmacher">
+<addon id="plugin.video.media-ccc-de" name="media.ccc.de" version="0.1.1" provider-name="Tobias Gruetzmacher">
   <requires>
     <import addon="xbmc.python" version="2.24.0"/>
-    <import addon="script.module.simplejson" version="3.3.0"/>
```
Comment (enen92) [inline plugin.video.media-ccc-de/addon.xml]: why did you remove simplejson? note that your addon is in the jarvis branch which means you cannot effectively rely on the json module in every kodi platform. Or does it only affect unit tests?

Comment (MartijnKaijser) [reply plugin.video.media-ccc-de/addon.xml]: you can certainly rely on json as it was in python2.5 and up. It just might be a bit slower

Comment (TobiX) [reply plugin.video.media-ccc-de/addon.xml]: I don't think the performance difference between simplejson and json is such a big deal and shouldn't everyone be on Python 2.7 nowadays (yeah, I know: hopes and dreams ;))?

Anyways, I was only using simplejson for tests and I rely on whatever requests is using (which might be simplejson AFAICS, but it also runs without) when parsing the "real" data.

Related: Are there statistics which Kodi versions are still in use? (e.g. addon repository meta data downloads split by user agent?)

Comment (MartijnKaijser) [reply plugin.video.media-ccc-de/addon.xml]: 3/4 is Krypton or up
